locallib.php

```diff
@@ -400,6 +400,10 @@ function zoom_get_user_id($required = true) {
                 $zoomuserid = $zoomuser->id;
                 $cache->set($USER->id, $zoomuserid);
             }
+            // If user does not have a Zoom account, throw an error.
+            if ($required && $zoomuser === false) {
+                throw new moodle_exception('zoomerr_usernotfound', 'mod_zoom');
```

Comment:
The `zoomerr_usernotfound` string expects a value to be provided for the link. In the other location where it is used, the link provided is `$config->zoomurl`. In this context, the config variable is not yet available, so perhaps:
```suggestion
                throw new moodle_exception('zoomerr_usernotfound', 'mod_zoom', '', get_config('zoom', 'zoomurl'));
```